modules/11_aerosols/exoGAINS/postsolve.gms

```diff
@@ -19,7 +19,7 @@ if((o_modelstat le 2),
 );
 
 *** Calculate AP emissions
-Execute "Rscript exoGAINSAirpollutants.R";
+Execute "Rscript --vanilla exoGAINSAirpollutants.R";
```

Comment:
I'll state to obvious (at least obvious to anyone who read the `R` man page): 
this deactivates the renv, making consistent results from REMIND runs a gamble once the versions of the packages required by `ecoGAINSAirpollutants.R` diverge between the renv library and whatever happens to be in `R_LIBS_SITE`.